frontend/src/layouts/ProtectedLayout.vue

```diff
@@ -6,31 +6,39 @@ import SideNav, {
     type ISideNavItem,
 } from '@/components/common/SideNav.vue';
 import sideNavData from '@/static/sideNav.json';
-import { FAM_APPLICATION_ID } from '@/store/Constants';
-import { isApplicationSelected, selectedApplicationId } from '@/store/ApplicationState';
-
+import { FAM_APPLICATION_ID, APP_ADMIN_ROLE } from '@/store/Constants';
+import {
+    isApplicationSelected,
+    selectedApplicationId,
+} from '@/store/ApplicationState';
+import LoginUserState from '@/store/FamLoginUserState';
 
 const navigationData = ref<[ISideNavData]>(sideNavData as any);
 
 // Show and hide the correct sideNav btn based on the application
-const setSideNavAddPermissionOption = () => {
-    if(selectedApplicationId.value === FAM_APPLICATION_ID) {
+const setSideNavOptions = () => {
+    if (selectedApplicationId.value === FAM_APPLICATION_ID) {
         disableSideNavOption('Add user permission', true);
         disableSideNavOption('Add application admin', false);
+        disableSideNavOption('Add delegated admin', true);
```

Comment:
Just a thought, Using the hardcoded string for many places isn't good. Probably need to be refactored later.

Reply:
it's a good suggestion maybe we can refactor it together with the sideNav to not use the json file